File: Source/Hubs/Bingo.Hubs/Player/Events/BingoSubmission/BingoSubmissionEvent.cs
using System.Collections.Generic;

namespace Pepp.Web.Apps.Bingo.Hubs.Player.Events.BingoSubmission
{
    public class BingoSubmissionEvent
    {
        /// <summary>
        /// The SignalR Hub Connection ID for this user.
        /// Used so that the Vote event is not sent to the user requesting the vote
        /// </summary>
        public string SubmitterConnectionID;
        /// <summary>
        /// The internal UserID of the user
        /// Used so the user cannot have multiple tabs open and vote for themselves
        /// </summary>
        public int UserID;
        /// <summary>
        /// The board tiles that compose the vote requestor's board at the time of bingo submission
        /// </summary>
        public List<TileDetail> BoardTiles;
    }
}

File: Source/WebService/ClientApp/src/app/shared/hubs/player/events/bingo-submission.event.ts
export class BingoSubmissionEvent {
  /*
   * The SignalR Hub Connection ID for this user.
   * Used so that the Vote event is not sent to the user requesting the vote
   */
  public submitterConnectionID: string;
  /*
   * The internal UserID of the user
   * Used so the user cannot have multiple tabs open and vote for themselves
   */
  public userID: number;
  /*
   * The board tiles that compose the vote requestor's board at the time of bingo submission
   */
  public boardTiles: TileDetail[];
}

class TileDetail
{
  /*
   * The text of the bingo tile
   */
  public text: string;
  /*
   * Bool flag indicating if tile is selected
   */
  public isSelected: boolean;
  /*
   * The row the bingo tile is associated with
   */
  public row: number;
  /*
   * The column the bingo tile is associated with
   */
  public column: number;
}
